util/mk-fipsmodule-cnf.pl

```diff
@@ -48,4 +49,10 @@
 tls1-prf-ems-check = $ems_check
 drbg-no-trunc-md = $drgb_no_trunc_dgst
 module-mac = $module_mac
+hkdf-key-check = $kdf_key_check
```

Comment:
Can you verify that the new fields are in both test/fipsmodule.cnf and providers/fipsmodule.cnf

Reply:
Yes.

Both files have these new fields as the following.

<details>
<summary>Click to show file contents</summary>

```
root@584034652cb0:/openssl/key-size-check# cat test/fipsmodule.cnf
[fips_sect]
activate = 1
install-version = 1
conditional-errors = 1
security-checks = 1
tls1-prf-ems-check = 1
drbg-no-trunc-md = 1
hkdf-key-check = 1
tls13-kdf-key-check = 1
tls1-prf-key-check = 1
sshkdf-key-check = 1
sskdf-key-check = 1
x963kdf-key-check = 1
module-mac = F5:73:E4:6E:9E:F1:44:29:5B:62:31:3C:0C:ED:1C:F5:45:67:79:86:B2:E3:4E:DE:21:49:5B:9F:02:55:EA:AF
root@584034652cb0:/openssl/key-size-check# cat providers/fipsmodule.cnf
[fips_sect]
activate = 1
conditional-errors = 1
security-checks = 1
tls1-prf-ems-check = 1
drbg-no-trunc-md = 1
module-mac = F5:73:E4:6E:9E:F1:44:29:5B:62:31:3C:0C:ED:1C:F5:45:67:79:86:B2:E3:4E:DE:21:49:5B:9F:02:55:EA:AF
hkdf-key-check = 1
tls13-kdf-key-check = 1
tls1-prf-key-check = 1
sshkdf-key-check = 1
sskdf-key-check = 1
x963kdf-key-check = 1
root@584034652cb0:/openssl/key-size-check#
```

</details>

Reply:
It was more of a "this is something you should check" when you do each of these indicator PR's :)